docs/tubemq_user_guide.md

```diff
@@ -1,4 +1,29 @@
 # TubeMQ User Guide
+## Prerequisites
+
+- Java 7/8(Java 9+ are not test)
+- Maven
+- [protoc 2.5.0](https://github.com/protocolbuffers/protobuf/releases/tag/v2.5.0)
+
+## Install protoc
+Install on windows operating system is simple:
+
+- Unzip `protoc-2.5.0-win32.zip`, will get `protoc.exe`.
+- Config environment variable, add path of `protoc.exe` to `Path` variable.
+
+Others os can following the protobuf official docuement.
+
+## Build source code
```

Comment:
Can we merge this section with the below one? `Build source code` and `Build` looks like duplicated. IMHO, It should be a supplement of the build section.

Maybe change it to
## Build
Go to the project root, and run
```bash
mvn clean package -Dmaven.test.skip
```
Each module of the project can also be build separately.

If want to build and debug source code in IDE, go to the project root, and run

```bash
mvn compile
```

This command will generate `.proto`  files corresponding Java source files(location is `target/generated-sources`).

When this command finished, you can use IDE import the project as maven project.

Reply:
Yes, at that time, I also think these two sections are a bit duplicated.Can the `Build` section divided into two child sections.One is `Build distribution tarball`, the others is `Build source code`, this is a bit clearer.

Reply:
Yes. I'm okay with that